cryptofeed/backends/backend.py

```diff
@@ -48,22 +48,22 @@ async def read_many_queue(self, count: int):
 
 class BackendBookCallback:
     async def __call__(self, *, feed: str, symbol: str, book: dict, timestamp: float, receipt_timestamp: float):
-        data = {'timestamp': timestamp, 'receipt_timestamp': receipt_timestamp, 'delta': False, BID: {}, ASK: {}}
+        data = {'feed': feed, 'symbol': symbol, 'timestamp': timestamp, 'receipt_timestamp': receipt_timestamp, 'delta': False, BID: {}, ASK: {}}
```

Comment:
that data is already present in the args passed to write. why do you think we need it here? Most the backends that use this will either ignore that or dump and store the json as-is, which would lead to a lot of duplicated entries (and wasted storage) since book data is so voluminous

Reply:
Because I the data doesn't include the symbol and I need that in callback function. Right now, I have to use 
`key.rsplit('-', 1)[-1]`
to get the symbol.

Other callbacks have feed & symbol in data. If we choose to get rid of duplicated data, shouldn't we remove them from all the callbacks?